src/components/clusterConfiguration/ClusterConfiguration.tsx

```diff
@@ -267,10 +267,10 @@ const ClusterConfiguration: React.FC<ClusterConfigurationProps> = ({ cluster })
                 type="submit"
                 name="save"
                 variant={ButtonVariant.secondary}
-                isDisabled={isSubmitting || !isValid || !dirty}
+                isDisabled={isSubmitting || !isValid}
                 onClick={submitForm}
               >
-                Validate & Save Changes
+                Validate {dirty ? '& Save Changes' : ''}
```

Comment:
IIUC there should be no case when user needs to revalidate without also having any changes. The cluster either gets updated from the server which automatically updates the state and validations or user has done some changes to the form and in that case `Validate & Save Changes` button is enabled.

I quite like how enabling that button lets user know there are unsaved changes (note that this is in addition reflected in the global errors in the toolbar)

This change also makes the button label dynamic which will be confusing for the user ("Sometimes I see 'Validate' sometimes I see 'Validate & Save', what does it mean?")

@andybraren your opinion?

Reply:
Sounds like consensus, so I removed the commit.